File: python/svix/models/application_out.py
# this file is @generated
import typing as t
from datetime import datetime

from .common import BaseModel


class ApplicationOut(BaseModel):
    uid: t.Optional[str] = None
    """Optional unique identifier for the application."""

    name: str
    """Application name for human consumption."""

    throttle_rate: t.Optional[int] = None
    """Maximum messages per second to send to this application.

    Outgoing messages will be throttled to this rate."""

    id: str
    """The Application's ID."""

    created_at: datetime

    updated_at: datetime

    metadata: t.Dict[str, str]

File: kotlin/lib/src/main/kotlin/models/ApplicationOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlin.time.Instant
import kotlinx.serialization.Serializable

@Serializable
data class ApplicationOut(
    /** Optional unique identifier for the application. */
    val uid: String? = null,
    /** Application name for human consumption. */
    val name: String,
    /**
     * Maximum messages per second to send to this application.
     *
     * Outgoing messages will be throttled to this rate.
     */
    val throttleRate: UShort? = null,
    /** The Application's ID. */
    val id: String,
    val createdAt: Instant,
    val updatedAt: Instant,
    val metadata: Map<String, String>,
)
